sass/components/_chart-controls.scss

```diff
@@ -7,12 +7,16 @@ cq-context {
         background: white;
         position: absolute;
         display: flex !important;
-        bottom: 37px !important;
+        bottom: 0px !important;
         left: 50%;
         padding: 0 10px;
         transform: translateX(-50%);
-        height: 32px;
-        width: auto;
+        height: 49px;
+        width: 100%;
+
+        flex-wrap: wrap; /* Optional. only if you want the items to wrap */
+        justify-content: center; /* For horizontal alignment */
+        align-items: center; /* For vertical alignment */
```

Comment:
`/* For horizontal alignment */` and `/* For vertical alignment */` are unnecessary comments.